EventsExpress.Db/EventsExpress.Db.csproj

```diff
@@ -19,5 +19,11 @@
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>    
+  <ItemGroup>
```

Comment:
Undo all stylecop usage. We have directory.build.props for this purpose.

Reply:
Done